### DISCUSSION TITLE
CTT View TranslateBrowsePath wrong error code

### DISCUSSION THREAD

**Discussion by johannwesely** [eclipse-milo/milo#1311] [General]
Hi i had the following issues

![image](https://github.com/user-attachments/assets/998e5773-57c8-4b35-96ec-dfc3e57494b3)

i changed the error code in BrowsePathsHelper Line 284  to
`     throw new UaException(StatusCodes.Bad_NodeIdUnknown);`

![image](https://github.com/user-attachments/assets/7a036b20-76dd-45ab-acef-c6859cd62bde)

now im Green can you check if this is correct solution or if i miss something in the if statement

**Comment by kevinherron** [comment]
I'll have to look at this closer and run all the tests. It could be correct, but then I wonder if line 225 also needs the same change.

**Reply by kevinherron** [reply]
#1315 and #1316
